config/crds/workloads_v1alpha1_console.yaml

```diff
@@ -35,14 +35,21 @@ spec:
             reason:
               type: string
             timeoutSeconds:
+              description:
+                Number of seconds that the console should run for. If the
+                process running within the console has not exited before this timeout
+                is reached, then the console will be terminated. Creation of a Console
+                will be rejected if the specified timeout exceeds the maximum timeout
+                defined within the ConsoleTemplate that the console is referring to.
```

Comment:
An alternative approach would be to just clamp the timeout to the max specified in the template. I don't really mind which we do - what do you think?

Reply:
Yes we'll in fact have to take that approach; because the ConsoleTemplate may not be present at the time that the Console is created. Updating this message!

Reply:
Well actually we don't *have* to, we could also throw an error in the reconcile loop and mark the console with an error status. But let's go with clamping